consent/strategy_default.go

```diff
@@ -697,7 +701,13 @@ func (s *DefaultStrategy) executeBackChannelLogout(ctx context.Context, r *http.
 	}
 
 	var wg sync.WaitGroup
-	hc := httpx.NewResilientClient()
+	hc := httpx.NewResilientClient(
+		httpx.ResilientClientWithClient(&http.Client{
+			Timeout: time.Minute,
+			Transport: &http.Transport{
+				TLSClientConfig: s.tlsClientConfig,
```

Comment:
Is it problematic if tlsClientConfig is nil?

Reply:
`tlsClientConfig` cannot be nil

https://github.com/ory/hydra/pull/2875/files#diff-265e72885dd37a9d8c6f81e2c928b2bfad035cecd98051ef2f04ea88a56ed540R77